CHANGELOG.md

```diff
@@ -77,10 +77,19 @@ Adding a new version? You'll need three changes:
   [#3832](https://github.com/Kong/kubernetes-ingress-controller/pull/3832)
 - Added license agent for Konnect-managed instances.
   [#3883](https://github.com/Kong/kubernetes-ingress-controller/pull/3883)
-- `Service`, `Route` and `Consumer` Kong entities now get assigned deterministic
+- `Service`, `Route`, and `Consumer` Kong entities now get assigned deterministic
```

Comment:
nit: I don't believe this change is correct

Reply:
That's what Grammarly has advised me to do 😅 According to the [src](https://www.grammarflip.com/curriculum/commas-items-in-a-series) it seems correct.